frontend/src/concepts/k8s/K8sNameDescriptionField/types.ts

```diff
@@ -26,9 +26,15 @@ export type K8sNameDescriptionFieldData = {
   };
 };
 
+export type K8sNameDescriptionType = {
+  displayName: string;
+  k8sName: string;
+  description?: string;
+};
```

Comment:
I think all of these should be optional as they are with a `K8sResourceCommon`.

Also, use `name` instead of `displayName` to align with `K8sNameDescriptionFieldData`.

```suggestion
export type K8sNameDescriptionType = {
  name?: string;
  k8sName?: string;
  description?: string;
};
```